### PR TITLE
removes king goat

### LINKED ISSUE #49522: Crusher and Cleaving Saw not working properly on Goat King Mega Fauna
Issue reported from  Round ID: 130802 (/tg/Station Bagil [ENGLISH] [US-WEST] [100% FREE LAG])

Reporting client version: 513.1511

Crusher properly marks goat king for detonation, but heiro trophy fails to proc. Cleaver also appeared to fail to cause bleed, making battle nigh impossible to do using melee weapons. Not looked through code, but assuming and hope this is just an oversight.

### LINKED ISSUE #50980: The dead goats in the goat king ruin aren't.
## Testmerges:

- #50558 
- #50955 

## Reproduction:

1. Find the lavaland goat king ruin.
2. Attempt to butcher the goat corpses.
3. Get attacked by the goat corpses.
  - This may aggro every goat corpse within viewing distance, I don't remember.

### LINKED ISSUE #51161: A consistent way to access the Wishgranter
hack

Reproduction: If you use a bluespace shelter capsule next to the Wishgranter Cube, it replaces the impossible to break walls with easy to break capsule walls. One can easily just break into the wishgranter from there by deconstructing one of the walls as demonstrated in the attached png. Bear in mind this was on Fulpstation so I don't know if it's been fixed or is due to the minor differences between tg Fulpcode.

### LINKED ISSUE #51256: Wizard den can be escaped from with shelter capsules
Title, mining shelter capsules can be brought to the wiz den and used to clear hyperspace tiles.

### Round ID: Not available. Local testing and confirmation done on the latest version of TG.
(Remember what I said about FUN?)

### Reproduction:

- Roll wizard.
- Buy a robeless spell apprentice (knock + fireball).
- Buy instant summons.
- Mark a small object and give it to your apprentice, have them  warp down to the mining lockers and open them with the knock spell
- Return them by having the apprentice drop the summon marked item in a locker, then also getting in the locker, when the recall spell is used it'll grab the whole locker with everything in it brought to you.
- Refund summon item and buy jaunt along with lesser summon guns or arcane barrage.
- Now with a shelter capsule use the magic mirror to turn yourself into an android (or any race that doesn't need air or pressure, but can still wear robes) and head to the south side of the den.
- Cast summon guns/barrage and fire at **all** the windows and engine parts until they're destroyed. 
- With the shelter capsule in hand, jaunt, then activate it and drop it during your jaunt on the space tiles where the den thrusters were. (Somehow the capsule drops outside your jaunt container)
FREEWIZ2
- Use mutate spell's hulk to smash the capsule open.
- Reach ~~FREEDOM~~ the Badmin target test dummy firing range.

Slightly simliar to #51161 due to how shelter capsules are overriding indestructible wall turfs in cases they really shouldn't.

EDIT: This could have potential for nuke-ops if they brought one back, further testing should be done to check if this works from inside the syndicate base.

> I find recalling apprentices back to the station using instant summons to be a feature, not a bug.

"I find it FUN."

### LINKED ISSUE #51352: KA deals weakened attacks and not normal attacks in King Goat arena.
Round ID: [138144](https://scrubby.melonmesa.com/round/138144)

130 weakened kinetic attacks split between King Goat and Emperor Goat form.

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

removes content from #48823 #49159 #50813
pretty much all king goat content is gone, things remaining are the ralsei plushie (now obtainable from the arcade instead of the real goats which did damage) and vortex recall not working in areas where you arent supposed to teleport
closes #49522 
closes #50980 
closes #51256 
closes #51352 
closes #51161 

## Why It's Good For The Game

king goat was a really uninteresting boss fight, running in circles and shooting with ka because if you get touched once you fall over and your limb breaks, stuff broke like shelter pods or pressure in the ruin, it spawned on the reserve z level every single round, the arcade goat plushies just rammed you when you won them as a prize, there is no joke in king goat, pretty sure the only reason its a goat is that its a port from yog where they have a goat fetish, badly resized sprites
i dont think king goat is worth having in the game

## Changelog
:cl:
del: Our Nanotrasen Elite Anti-Animal Team hunted down King Goat and stopped his reign forever. As a memorial of this event you can now get a cute goat plushie from the arcades.
/:cl:

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

File: code/game/turfs/open/_open.dm [hunk 1]
```diff
@@ -78,22 +78,6 @@
 	heavyfootstep = FOOTSTEP_LAVA
 	tiled_dirt = FALSE
 
-/turf/open/indestructible/carpet
```
Comment (bobbahbrown) [inline code/game/turfs/open/_open.dm:81]: is this used anywhere else

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -323,9 +323,7 @@
 	var/teleporting = FALSE
 
 /obj/item/warp_cube/attack_self(mob/user)
-	var/turf/current_location = get_turf(user)
```
Comment (bobbahbrown) [inline code/modules/mining/lavaland/necropolis_chests.dm:326]: does any other ruin/area rely on this to prevent teleporting

Comment (Fikou) [reply code/modules/mining/lavaland/necropolis_chests.dm:326]: it was added by king goat pr so probbablably not

File: code/__DEFINES/achievements.dm [hunk 3]
```diff
@@ -42,7 +42,6 @@
 #define BOSS_MEDAL_LEGION	  "Legion Killer"
 #define BOSS_MEDAL_TENDRIL	  "Tendril Exterminator"
 #define BOSS_MEDAL_SWARMERS   "Swarmer Beacon Killer"
-#define BOSS_MEDAL_KINGGOAT   "King Goat Killer"
```
Comment (AnturK) [inline code/__DEFINES/achievements.dm:45]: Leave the achievements in for now, i'll have to add some code to make removing them from metadata more graceful

**Review by AnturK — CHANGES_REQUESTED** [review]

**Comment by wesoda25** [comment]
I got to centcomm through this once

**Comment by HomocidlePancakes** [comment]
Probably an admin killed it on CC and spawned a ladder.

**Comment by Acer202** [comment]
I dareday I'll miss our forever-dead girl Lyna Lavas more than the goats.

Good change; it really was no more than a storm stopover for many.

**Comment by nemvar** [comment]
The king is dead, long live the king!

**Comment by adamkad1** [comment]
Goat plushies were funny to put in hallways though

**Pull request merged**